frontend/src/metabase-types/api/table.ts

```diff
@@ -1,4 +1,4 @@
-import type { Card } from "./card";
+import type { Card, CardType } from "./card";
```

Comment:
1. New -> SQL query -> SELECT * FROM ORDERS -> Save as SQL query -> Run
2. Click on "Explore results"

The icon in query builder header is now `database` - is this expected?
If you go to the editor the data source will have `table2` icon.

3. Click on a cell -> Filter -> Click on an operator -> You should get an ad-hoc query

The icon in query builder header is still `database` - is this expected?
If you go to the editor the data source will have `table2` icon.